invoke.go

```diff
@@ -88,3 +91,21 @@ func invoke(ctx context.Context, call APICall, settings CallSettings, sp sleeper
 		}
 	}
 }
+
+func grpcCtxOpts(ctx context.Context) []grpc.CallOption {
```

Comment:
I'm not sure how to test this function. `grpc.FaiFast` return functions, so we can't compare the return value to see if we get what we expect.